geoips/commandline/geoips_list.py

```diff
@@ -275,10 +319,15 @@ def list_implemented_interfaces(self, package_name):
             interface_data = []
             for interface_type in pkg_registry.keys():
                 for interface_name in pkg_registry[interface_type].keys():
-                    interface_data.append(
-                        [plugin_package_name, interface_type, interface_name]
-                    )
-            headers = ["GeoIPS Package", "Interface Type", "Interface Name"]
+                    interface_entry = []
+                    for header in list(headers.keys()):
+                        if header == "package":
+                            interface_entry.append(plugin_package_name)
+                        elif header == "plugin_type":
+                            interface_entry.append(interface_type)
+                        elif header == "interface":
+                            interface_entry.append(interface_name)
+                    interface_data.append(interface_entry)
```

Comment:
It seems like there is a lot of repetition of this kind of thing. Could this be centralized somewhere in a single function that can be called with the requested columns?

Reply:
This will likely need a lot of refactoring and a lot of file changes. I'll make an issue for this one.

Reply:
See #602.